src/eradiate/scenes/surface/_dem.py

```diff
@@ -0,0 +1,266 @@
+from __future__ import annotations
+
+import typing as t
+
+import attr
+import numpy as np
+import pint
+import xarray as xr
+
+from ._core import Surface, surface_factory
+from ..bsdfs import BSDF, LambertianBSDF, bsdf_factory
+from ..core import KernelDict
+from ..shapes import BufferMeshShape, FileMeshShape, Shape, shape_factory
+from ...attrs import documented, get_doc, parse_docs
+from ...contexts import KernelDictContext
+from ...units import to_quantity
+from ...units import unit_context_config as ucc
+from ...units import unit_registry as ureg
+from ...util.misc import onedict_value
+
+
+@surface_factory.register(type_id="dem")
+@parse_docs
+@attr.s
+class DEMSurface(Surface):
+    """
+    DEM surface [``dem``]
+
+    A surface description for digital elevation models. This object holds only the Shape.
+    The associated BSDF is defined within the Shape plugin.
+
+    This surface supports instantiation based on preprocessed geotiff files,
+    triangulated meshes in the PLY and OBJ formats and analytical surfaces based on functions
+    which map x and y coordinates to an elevation value.
+
+    For details refer to the corresponding constructor methods.
```

Comment:
Please rather list class method constructors like in the ``MolecularAtmosphere`` docstring.